javascript/javascript1/week2/candy.js

```diff
@@ -0,0 +1,51 @@
+let candyPrice = 0;
+let boughtCandyPrices = [];
+
+function addCandy(candyType, weight) {
+    if (candyType === "Sweet") {
+        candyPrice = weight * 0.5;
+        boughtCandyPrices.push(candyPrice);
+    } else if (candyType == "Chocolate") {
+        candyPrice = weight * 0.7;
+        boughtCandyPrices.push(candyPrice);
+    } else if (candyType == "Toffee") {
+        candyPrice = weight * 1.1;
+        boughtCandyPrices.push(candyPrice);
+    } else if (candyType == "Chewing-Gum") {
+        candyPrice = weight * 0.03;
+        boughtCandyPrices.push(candyPrice);
+    }
```

Comment:
You can write this `boughtCandyPrices.push(candyPrice);` one time after all of your if-else conditions instead of repeating it in each one